src/lib/core/components/visualizations/VisualizationsContainer.tsx

```diff
@@ -242,7 +243,12 @@ function NewVisualizationPicker(props: Props) {
       </div>
       <h3>Select a visualization</h3>
       <Grid>
-        {visualizationsOverview.map((vizOverview, index) => {
+        {/* orderBy ensures that available visualizations render ahead of those in development */}
+        {orderBy(visualizationsOverview, [
+          function (viz) {
+            return viz.name ? visualizationTypes[viz.name] : undefined;
+          },
```

Comment:
```suggestion
          (viz) => viz.name ? visualizationTypes[viz.name] : undefined
```

This has fewer keywords and more hieroglyphics but I still think it's readable.